setup.py

```diff
@@ -44,19 +70,30 @@ def get_version(module='spyder_kernels'):
 setup(
     name='spyder-kernels',
     version=get_version(),
-    keywords='spyder jupyter kernel',
+    keywords='spyder jupyter kernel ipython console shell',
```

Comment:
Please remove `shell` from here. This is not something like `Xonsh`.

Reply:
Sorry, removed.